_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,310 @@
+  video][london vid], [Presentation video][revault vid])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- **Bitcoin Core 0.20.0** has been tagged and will likely be released
+  around the same time this newsletter is published.  We'll describe
+  this new major release in more detail next newsletter.
+
+- [LND 0.10.1-beta.rc3][] is the latest release candidate for the next
+  maintenance release of LND.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #19010][] and [Bitcoin Core #19044][] are the third and fourth
+  steps, respectively, of a [series of five pull requests][Bitcoin Core #18876]
+  towards support for serving [compact block filters][topic compact block
+  filters] on the P2P network, as specified in [BIP157][]. The first step was
+  covered in [Newsletter #98][news98 bitcoin core 18877].
+
+    With these changes, nodes that enable the compact block filter index with the
+    `-blockfilterindex` configuration parameter can now respond to `getcfcheckpt`,
+    `getcfheaders`, and `getcfilters` requests with the corresponding `cfcheckpt`,
+    `cfheaders`, and `cfilters` responses. The node does not yet advertise support
+    for BIP157 with `NODE_COMPACT_FILTERS` in its version message. The final step,
+    [Bitcoin Core #19070][], is under review at the time of this writing and
+    would enable nodes to signal the ability to serve compact block filters.
+    The feature is disabled by default and can be enabled with the
+    `-peerblockfilters` configuration parameter.
+
+- [Bitcoin Core #16939][] changes how long Bitcoin Core waits until
+  it queries DNS seeds for the IP addresses of potential peers.
+  Previously, if the node had peer IP addresses in its database, it
+  would try opening several connections and wait 11 seconds for
+  successful connections before requesting new addresses.  Now, if it
+  has more than 1,000 IP addresses in its database---which is common for
+  nodes that have been online more than a few hours---it'll wait up to 5
+  minutes before querying.  This improves the chance that a restarted
+  node will entirely use P2P address discovery without relying on the
+  centralized DNS seeds.
+
+- [LND #4228][] adds a new wallet command `labeltx` for labelling past onchain
```

Comment:
Grepped for our use of labeling and labelling in the code and found this to be the first instance of the non-American variant and three instances in the RBF in the Wild post.

```suggestion
- [LND #4228][] adds a new wallet command `labeltx` for labeling past onchain
```